src/libs/ajax.js

```diff
@@ -35,17 +35,6 @@ import { getConfig } from 'src/libs/config';
 import { getTerraUser } from 'src/libs/state';
 import * as Utils from 'src/libs/utils';
 
-window.ajaxOverrideUtils = {
-  mapJsonBody: _.curry((fn, wrappedFetch) => async (...args) => {
```

Comment:
Currying wasn't really necessary here. These are always called with one argument, then `withInstrumentation` passes the `wrappedFetch` function.